users/models.py

```diff
@@ -369,6 +377,54 @@ def get_all(cls):
         """
         return cls.active_users.all()
     
+    def confirm_pending_email(self):
```

Comment:
Add @transaction.atomic decorator for clarity, rather than using with transaction.atomic() inside the method.